application/conf/config.json

```diff
@@ -8,104 +8,105 @@
         "magnitude": 1,
         "config": {}
       },
-	  {
```

Comment:
Here the "default" is 2 spaced indented.
And your line is 1 tab and 2 spaces indented.
I think, it's reasonable to stick to 2 spaces indentation for this file.